tools/update_roles.php

```diff
@@ -0,0 +1,669 @@
+<?php
+/**
+ * Script to add or remove permissions from a role
```

Comment:
Is there a frontend to manage the role permissions too, or do you need access to the shell to do it?

Reply:
There is no front end right now - but hopefully that can go in the config module soon!